packages/client-search/src/types/ApiKey.ts

```diff
@@ -0,0 +1,14 @@
+export type ApiKeyACL =
```

Comment:
You can you create a file named `ApiKeyAclType` with contents similar too: https://github.com/algolia/algoliasearch-client-javascript/blob/master/packages/client-search/src/types/BatchActionType.ts?